packages/components/src/components/progress-slider-mobile/progress-slider-mobile.tsx

```diff
@@ -1,11 +1,22 @@
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
 import { getTimePercentage } from '@deriv/shared';
-import ProgressTicksMobile from './progress-ticks-mobile.jsx';
+import ProgressTicksMobile from './progress-ticks-mobile';
 import CircularProgress from '../circular-progress';
 import RemainingTime from '../remaining-time';
 import Text from '../text';
+import { TGetCardLables } from '../types';
+
+type TProgressSliderMobileProps = {
+    className?: string;
+    current_tick: number;
+    expiry_time: number & string;
```

Comment:
did you mean this? 
```suggestion
    expiry_time: number | string;
```

Reply:
Hello @shayan-deriv, might I ask why you are suggestion this?

Reply:
@shayan-deriv, actually based on the `Deriv-API` types, this is a `number`, so I'll just update it to that.